base/compiler/utilities.jl

```diff
@@ -337,6 +337,16 @@ function foreachssa(@specialize(f), @nospecialize(stmt))
     end
 end
 
+function foreachanyssa(@specialize(f), @nospecialize(stmt))
```

Comment:
```suggestion
function foreach_anyssa(@specialize(f), @nospecialize(stmt))
```

maybe?

Reply:
Probably makes sense just to spell these as `foreach(f, anyssas(stmt))` and `foreach(f, ssas(stmt))` rather than making separate functions.

Reply:
Actually, scratch that. The compiler isn't currently good enough to do that without allocations. I'll just do what you suggested here.